wire-ios/Wire-iOS/Sources/UserInterface/Settings/Debug Report/SettingsDebugReportViewController.swift

```diff
@@ -0,0 +1,132 @@
+
+    // MARK: - Types
+
+    private typealias Strings = L10n.Localizable.Self.Settings
+
+    // MARK: - Properties
+
+    private let viewModel: SettingsDebugReportViewModelProtocol
+
+    // MARK: - Views
+
+    private let infoLabel: UILabel = {
+        let label = UILabel()
+        label.text = Strings.TechnicalReport.info
+        label.textColor = SemanticColors.Label.textDefault
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var sendReportButton: UIButton = {
+        return createButton(
+            title: Strings.TechnicalReport.sendReport.capitalized,
+            action: #selector(didTapSendReport)
+        )
+    }()
+
+    private lazy var shareReportButton: UIButton = {
+        return createButton(
+            title: Strings.TechnicalReport.shareReport.capitalized,
+            action: #selector(didTapShareReport)
+        )
+    }()
+
+    // MARK: - Life cycle
+
+    init(viewModel: SettingsDebugReportViewModelProtocol) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = SemanticColors.View.backgroundDefault
+        navigationItem.setDynamicFontLabel(title: Strings.TechnicalReportSection.title.capitalized)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    // MARK: - Setup
+
+    private func setupViews() {
+        view.addSubview(infoLabel)
+        view.addSubview(sendReportButton)
+        view.addSubview(shareReportButton)
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            infoLabel.topAnchor.constraint(equalTo: view.safeTopAnchor, constant: 20),
+            infoLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            infoLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+
+            shareReportButton.bottomAnchor.constraint(equalTo: view.safeBottomAnchor, constant: -30),
+            shareReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            shareReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            shareReportButton.heightAnchor.constraint(equalToConstant: 48),
+
+            sendReportButton.bottomAnchor.constraint(equalTo: shareReportButton.topAnchor, constant: -8),
+            sendReportButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            sendReportButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            sendReportButton.heightAnchor.constraint(equalToConstant: 48)
```

Comment:
I would recommend not setting a height for the buttons because that would block from the buttons to expand when the font size grows

Reply:
done https://github.com/wireapp/wire-ios/pull/1658/commits/8c85d9ddbe3d4f96ae06fb91a9db48865e963819